pkg/asset/templates/kube-apiserver.yaml

```diff
@@ -23,26 +23,25 @@ spec:
         - /hyperkube
         - apiserver
         - --bind-address=0.0.0.0
+        - --secure-port=443
+        - --insecure-port=8080
         - --advertise-address=$(MY_POD_IP)
-        - --etcd-servers={{ .ETCDServers }}
+        - --etcd-servers={{ range $i, $e := .EtcdServers }}{{ if $i }},{{end}}{{ $e }}{{end}}
```

Comment:
Just noting that I want to move this logic out of the template soon. I didn't put in a #TODO because it ends up in the rendered templates too.